org.osgi.test.cases.cm/src/org/osgi/test/cases/cm/junit/CMCoordinationTestCase.java

```diff
@@ -345,4 +372,8 @@ public void updated(Dictionary<String, ? > properties)
 		assertFalse(events.get(3));
 
 	}
+
```

Comment:
Why do we have this?

Reply:
sorry i reset the commit. so i do not know what happened here

Reply:
I was asking about the getLongProperty method you had added as a stub method, but it is gone now.